ando/checker.py

```diff
@@ -20,6 +20,24 @@
 
 dir_rules = os.path.join(os.path.dirname(__file__)) + 'rules/'
 def main():
+    """
+     
+    Main file of the validator. uses other class methods for checking
+    different aspects of the directory path.
+
+    Now uses same function for verbose and not verbose mode 
```

Comment:
this sentence should not be in the docstring...

Reply:
Deleted